docs/pages/enroll-resources/database-access/enroll-aws-databases/aws-docdb.mdx

```diff
@@ -0,0 +1,235 @@
+click **Create Role** to complete the process.
+
+## Step 4/6. Start the Database Service
+
+(!docs/pages/includes/start-teleport.mdx service="the Database Service"!)
+
+The Database Service will proxy the Amazon DocumentDB cluster with the URI you
+specified earlier. Keep in mind that AWS IAM changes may not propagate
+immediately and can take a few minutes to come into effect.
+
+## Step 5/6. Create a DocumentDB IAM user
+
+In this step, we will create a DocumentDB IAM user that can be accessed by
+Teleport. 
+
+### Create an IAM role
+
+Navigate back to the Roles page on the AWS Web Console and create a new role.
+Select the "AWS account" option, which creates a default trust policy to allow
+other entities in this account to assume this role:
+
+![Create Role Step 1](../../../../img/database-access/guides/dynamodb-create-role-1.png)
+
+Skip the "Add Permissions" page by clicking "Next", and give the role a name.
+In this guide, we will use the example "teleport-docdb-user" for this role.
+
+Now click **Add new tag** at Step 3, use `TeleportDatabaseService` for the key
+and `Allowed` for the value. Then click **Create Role** to complete the process.
+
+![Create Role Step 3](../../../../img/database-access/guides/aws-create-role-add-tags.png)
+
+### Create a DocumentDB user
+
+Login your DocumentDB cluster with your master username and password from a
+machine that has network access to the DocumentDB cluster. Create a DocumentDB
+user with the IAM role ARN and use the `MONGODB-AWS` mechanisms for
+authentication:
+```code
+use $external;
+db.createUser(
+    {
+        user: "<Var name="arn:aws:iam::(=aws.aws_account_id=):role/teleport-docdb-user"/>",
+        mechanisms: ["MONGODB-AWS"],
+        roles: [ { role: "root", db: "admin" },  ]
+    }
+);
+```
+
+This example creates a DocumentDB IAM user with root privileges. You can adjust
+the privileges as required.
+
+<Admonition type="tip" title="Avoid password authentication">
+This step is a one-time setup that must use password authentication. However,
+after creating the first IAM user, you can skip password authentication by
+accessing DocumentDB through Teleport and creating more DocumentDB IAM users
+using the first IAM user.
+</Admonition>
+
+## Step 6/6 Connect
+
+Once the Database Service has started and joined the cluster, log in to see the
+registered databases:
+
+```code
+$ tsh login --proxy=<Var name="example.teleport.sh:443" /> --user=alice
+$ tsh db ls
+# Name     Description Labels
+# -------- ----------- --------
+# my-docdb             env=dev
+```
+
+Retrieve credentials for the database and connect to it as the `teleport-docdb-user` user:
+
+```code
+$ tsh db connect --db-user=teleport-docdb-user --db-name=test my-docdb
+```
+
+Log out of the database and remove credentials:
+
+```code
+$ tsh db logout rds-example
+```
+
+## Troubleshooting
+
+(!docs/pages/includes/database-access/aws-troubleshooting.mdx!)
```

Comment:
This isn't the fault of this PR, but we should really split out or template the db specific troubleshooting.
I was surprised to see certificate error troubleshooting for RDS in the rendered preview.

Does DocumentDB have this issue? I assume there's some CA bundle for it, but I've never looked at DocumentDB before.

Reply:
Good catch. DocumentDB uses the same bundle as RDS. I will update the error description there to include DocumentDB.